src/components/pages/ImageClassification/useSelectNewAttribute.js

```diff
@@ -1,6 +1,7 @@
 import { useState } from 'react'
 import { getBenthicOptions } from '../../../library/getOptions'
 import { toast } from 'react-toastify'
+import { useHttpResponseErrorHandler } from '../../../App/HttpResponseErrorHandlerContext'
```

Comment:
Nit pick. Keep the 3rd party imports separate from imports from the application

Reply:
Agree stylistically. :)